apps/sms/views/conversation/js/information.js

```diff
@@ -247,14 +248,17 @@ var VIEWS = {
 
         ConversationView.promptContact({
           number: target.dataset.number
-        });
+        }).then(
+          () => Navigation.toPanel('thread', { id: this.id })
+        );
```

Comment:
__unrelated question:__ was it UX requirement to leave participants view when user switches to another app to continue action (call, send email)? Looks like all this promise stuff inside prompts (that is never resolved in this case btw) just to support this case.